Bottom: 641722d2639e77b671a09ebef02278befe639750
Top:    424afdaa5d93e698f2fafc31c659dd2d709b81c7
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 18:21:25 +0000

Refresh of wip-on-quad-edge-play

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 35f2330..0c3ad02 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -294,7 +294,7 @@ fprint_idlist (FILE *f, int *ids, int num_ids)
 }
 
 void
-object3d_export_to_step (object3d *object, char *filename)
+object3d_export_to_step (object3d *object, const char *filename)
 {
   FILE *f;
   time_t currenttime;
diff --git a/src/hid/common/object3d.h b/src/hid/common/object3d.h
index 8d081a2..9a34fa8 100644
--- a/src/hid/common/object3d.h
+++ b/src/hid/common/object3d.h
@@ -17,5 +17,5 @@ void object3d_add_vertex (object3d *object, vertex3d *vertex);
 void object3d_add_face (object3d *object, face3d *face);
 object3d *object3d_create_test_cube (void);
 object3d *object3d_from_board_outline (void);
-void object3d_export_to_step (object3d *object, char *filename);
+void object3d_export_to_step (object3d *object, const char *filename);
 void object3d_test_board_outline (void);
